pyglmnet/datasets.py

```diff
@@ -0,0 +1,35 @@
+"""
+A set of convenience functions to download datasets for illustrative examples
+"""
+import subprocess
+import pandas as pd
+
+
+def fetch_tikhonov_data(dpath='/tmp/glm-tools'):
+    cmd = 'rm -rf ' + dpath
+    subprocess.call(cmd, shell=True)
+    cmd = 'mkdir ' + dpath
+    subprocess.call(cmd, shell=True)
+
+    fname = dpath + '/' + 'fixations.csv'
+    url = ("https://raw.githubusercontent.com/"
+           "glm-tools/datasets/master/tikhonov/fixations.csv")
+    cmd = 'curl -o ' + fname + ' ' + url
+    subprocess.call(cmd, shell=True)
+    fixations_df = pd.read_csv(fname)
+
+    fname = dpath + '/' + 'probes.csv'
```

Comment:
use `op.join` to be operating system agnostic. We should have an appveyor builder for checking Windows compatibility too ..